### DISCUSSION TITLE
Shouldn't coupons be auto-expired?

### DISCUSSION THREAD

**Discussion by lat9** [zencart/zencart#6563] [Ideas]
Banners, Specials and Featured products are auto-expired ... why aren't coupons?

**Comment by drbyte** [comment]
After checking whether the coupon is enabled, it also checks the start-date and end-date of the coupon:
https://github.com/zencart/zencart/blob/0822b02b4e17c83461017656eb6f3e100e43e347/includes/modules/order_total/ot_coupon.php#L398-L408

Is there a situation where this isn't sufficient?

**Comment by lat9** [comment]
I was thinking more in the admin interface.
